packages/appstore/src/components/modals/account-disabled-modal/account-disabled-modal.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer } from 'mobx-react-lite';
+import { useStore } from '@deriv/stores';
+
+const AccountDisabledModal = () => {
```

Comment:
let's specifically mention about cfd account to prevent confusion with a Deriv account.